guides/v2.1/release-notes/ReleaseNotes2.1.16CE.md

```diff
@@ -0,0 +1,212 @@
+
+
+<!--- ENGCOM-2975 -->* The dropdown menu of available countries that is accessible throughout both storefront and Admin is now populated as expected. *Fix submitted by [Nilesh Lokhande](https://github.com/nilesh2jcommerce) in pull request [17194](https://github.com/magento/magento2/pull/17194)*. [GitHub-2146](https://github.com/magento/magento2/issues/2146)
+
+
+
+
+### Infrastructure
+
+<!--- MAGETWO-92836 -->* Magento 2.1.16 now provides support for PHP 7.1. 
+
+
+<!--- ENGCOM-2730 -->* You can now use REST to add a configurable product to a shopping cart without creating a duplicate product entry. *Fix submitted by [Pratik Oza](https://github.com/mage2pratik) in pull request [17476](https://github.com/magento/magento2/pull/17476)*.  [GitHub-15028](https://github.com/magento/magento2/issues/15028)
+
+<!--- ENGCOM-2696 -->* The sidebars for the wishlist on the catalog, my account, and checkout pages now render special characters correctly. Previously, the browser displayed `&trade;` instead of rendered special characters on these pages. *Fix submitted by [Pratik Oza](https://github.com/mage2pratik) in pull request [17422](https://github.com/magento/magento2/pull/17422)*.
+
+
+
+### Password
+
+<!--- ENGCOM-2653 -->* The password reset strength meter that Magento displays when a customer resets a password now works as expected. *Fix submitted by [Jignesh Baldha](https://github.com/jignesh-baldha) in pull request [17290](https://github.com/magento/magento2/pull/17290)*. [GitHub-13429](https://github.com/magento/magento2/issues/13429)
+
+### Reports
+
+<!--- ENGCOM-2764 -->* The Year-to-date dropdown accessed from **Stores** > **Configuration** > **General** > **Reports** > **Dashboard** now displays a numerical list that ranges from 01 to 12 as expected. *Fix submitted by [Ronak Patel](https://github.com/ronak2ram) in pull request [17496](https://github.com/magento/magento2/pull/17496)*. [GitHub-17289](https://github.com/magento/magento2/issues/17289)
+
+
+### Review
+
+<!--- ENGCOM-2806 -->* Magento now displays a `404 page not found` error when a customer tries to navigate to a product review that is not accessible. Previously. Magento displayed a PHP error code. *Fix submitted by [Pratik Oza](https://github.com/mage2pratik) in pull request [17632](https://github.com/magento/magento2/pull/17632)*. [GitHub-13102](https://github.com/magento/magento2/issues/13102)
+
+
+
+
+
+###  Sales
+
+<!--- ENGCOM-2933 -->* Join extension attributes are now added to results for orders created through REST. Previously, the JSON response did not contain `some_id` in the extension attributes property. *Fix submitted by [Sam Butler Thompson](https://github.com/Scarraban) in pull request [16169](https://github.com/magento/magento2/pull/16169)*. [GitHub-8035](https://github.com/magento/magento2/issues/8035)
```

Comment:
Rewrite to

When you define an extension attribute for the OrderInterface with a join table, the specified attribute code value is now returned as expected in REST calls.